webservices/tasks/legal_docs.py

```diff
@@ -129,30 +145,11 @@ def weekly_reload_all_aos():
     """
     logger.info(" Weekly (%s) reload of all AOs ", datetime.date.today().strftime("%A"))
     load_advisory_opinions()
-    logger.info(" Weekly (%s) reload of all AOs completed", datetime.date.today().strftime("%A"))
-    slack_message = "Weekly reload of all AOs completed in {0} space".format(get_app_name())
+    logger.info(" Weekly (%s) reload of all AOs successfully", datetime.date.today().strftime("%A"))
```

Comment:
Could you updated the log message on line#148 & line#149 to `Weekly (%s) reload of all AO(s) completed successfully`